core/index.py

```diff
@@ -670,11 +670,12 @@ def get_page_text(page):
    no_text = ["Text not available"]
    solr = SolrConnection(settings.SOLR)
    title, date = page.issue.title, page.issue.date_issued
-   query = 'title:"' + str(title) + '" and date:' + str(date).replace('-', '') + \
-           ' and sequence:' + str(page.sequence)
+   query = 'title: %s and date:%s and sequence:%s' % (str(title).replace('"', ''),
```

Comment:
Does solrpy not have a function to escape query components? Based on [https://lucene.apache.org/core/2_9_4/queryparsersyntax.html#Escaping%20Special%20Characters](the Lucene documentation) there are other characters which we should escape as well. If Solrpy doesn't have a function which does that, we could add a little utility.

Reply:
My bad, there is actually solr_escape that I just used

Reply:
Thank you